plugins/sep/colorconfig/CustomColorOperations.hh

```diff
@@ -0,0 +1,50 @@
+//
+// Created by jelle on 02/06/2021.
+//
+
+#ifndef SCROOMCPPPLUGINS_CUSTOMCOLOROPERATIONS_HH
+#define SCROOMCPPPLUGINS_CUSTOMCOLOROPERATIONS_HH
+
+#include <boost/shared_ptr.hpp>
+#include <scroom/pipettelayeroperations.hh>
+#include <scroom/layeroperations.hh>
+#include "CustomColor.hh"
+
+
+class PipetteCommonOperationsCustomColor
+        : public PipetteLayerOperations
+                , public CommonOperations
+{
+protected:
+    int bps;
+    int spp;
+    std::vector<CustomColor*> colors;
+
+
+public:
+    using Ptr = boost::shared_ptr<PipetteCommonOperationsCustomColor>;
+
+public:
+    PipetteCommonOperationsCustomColor(int bps_)
+            : bps(bps_){
+        spp = 0;
+    };
+    void                 setSpp(int samplesPerPixel);
```

Comment:
Why is it that `bps` is set in the constructor, and `spp` in a setter? Shouldn't the two be treated the same?

Reply:
Changed constructor to take this task